distribution/osbuild-worker@.service

```diff
@@ -1,5 +1,6 @@
 [Unit]
 Description=OSBuild Composer Worker (%i)
+Requires=osbuild-composer.socket
```

Comment:
Is it ok to depend on both sockets here?

Reply:
We could split it up, but I don't see why it would not be ok?